sdk/package.json

```diff
@@ -18,7 +18,6 @@
     "fs": "0.0.2",
     "node-uuid": "^1.4.3",
     "q": "^1.4.1",
-    "superagent": "git://github.com/visionmedia/superagent.git#45a1290b2fe5a56a1d77ca74e0b236178d58d848",
-    "try-json": "^1.0.0"
+    "superagent": "git://github.com/visionmedia/superagent.git#45a1290b2fe5a56a1d77ca74e0b236178d58d848"
```

Comment:
Yes, although it might be good to do it on a non-breaking branch, as this one won't go in for a while.